portal/styles/button.scss

```diff
@@ -4,4 +4,8 @@
   text-decoration: underline;
   cursor: pointer;
   color: $govuk-link-colour;
+  padding: 0;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
```

Comment:
I expected we'd want `inherit` for these

Reply:
This change is actually from trying to merge the `govuk-link` class into this class. @oscar-richardson-softwire's going to try to make that more maintainable with SASS features